packages/components/src/components/as-range-slider/MouseTrack.ts

```diff
@@ -22,7 +22,11 @@ export class MouseTrack {
 
   private _handleRelease(eventProperties: MouseEvent, listeners: MouseListeners, customListeners: MouseListeners) {
     document.removeEventListener('mousemove', listeners.move);
+    document.removeEventListener('touchmove', listeners.move);
     document.removeEventListener('mouseup', listeners.release);
+    document.removeEventListener('touchend', listeners.release);
+
+    document.removeEventListener('dragstart', this.preventAndStop);
```

Comment:
why this listener is not in the listeners object?

Reply:
Because we don't have any custom references coming from components. It is always added with the same function reference.